rescript/screenseq.py

```diff
@@ -40,7 +40,11 @@ def cull_seqs(sequences: DNAIterator, num_degenerates: int = 5,
 
     n_jobs = cpu_count()
```

Comment:
We'll want to include `n_jobs` as a user defined value, with a default of 1, and make it accessible through `plugin_setup.y`, [here](https://github.com/bokulich-lab/RESCRIPt/blob/07ceb370663999a9c6c7e279c8a5855477baa33c/rescript/plugin_setup.py#L386).

Reply:
Added!